src/agent/structures/structures.ts

```diff
@@ -98,21 +105,17 @@ export interface ResumeChatParameters {
 }
 
 export interface TransferChatParameters {
-  target?: TransferTargetGroup | TransferTargetAgent;
+  id: string;
+  target?: TransferTarget;
```

Comment:
```suggestion
  target?: TransferTargetGroup | TransferTargetAgent;
```
To be reverted. These 2 lines were correct before:

1. `id` field shouldn't be in this interface because the chat id is passed as a separate parameter `id` to the `transferChat` method:
https://github.com/livechat/lc-sdk-js/blob/8f02f0dfa523a01d037183eb6177288e94830b67/src/agent/rtm.ts#L178

2. `TransferTargetGroup` and `TransferTargetAgent` interfaces have different type of `ids` - agent ids are strings and group ids are numbers. They should both be reverted